gsy_framework/serialization.py

```diff
@@ -0,0 +1,17 @@
+import pickle
+from typing import Dict
+from zlib import compress, decompress
+
+
+class CompressSerialization:
+    """Custom Serializer / Deserializer."""
+    @staticmethod
+    def encode_and_compress_dict(uncompressed_dict):
```

Comment:
Nit: we try to use type hints to document what kind of input we have. For example, in this case it would be:
```suggestion
from typing import Dict
# ...
    def encode_and_compress_dict(uncompressed_dict: Dict) -> bytes:
```
The same would apply to other methods below